lms/envs/aws.py

```diff
@@ -176,6 +176,7 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
+ENTERPRISE_API_URL = ENV_TOKENS.get('ENTERPRISE_API_URL', LMS_ROOT_URL + '/enterprise/api/v1/')
```

Comment:
@kevinchugh @staubina @bradenmacdonald The PR with the fix is here: https://github.com/edx/edx-platform/pull/14610 :)